lib/api/v3/work_packages/schema/work_package_schema_representer.rb

```diff
@@ -117,6 +117,18 @@ def initialize(schema, self_link:, **context)
                  type: 'Formattable',
                  required: false
 
+          schema :duration,
+                 type: 'Duration',
+                 required: false,
+                 writable: false,
+                 show_if: ->(*) { !represented.milestone? && OpenProject::FeatureDecisions.work_packages_duration_field_active? }
+
+          # TODO: Remove this, once the duration feature is complete
+          # This enables hiding the duration field on the form_configuration
+          ::Type.add_constraint :duration, ->(*) {
+            OpenProject::FeatureDecisions.work_packages_duration_field_active?
+          }
```

Comment:
We can simply always hide the duration property. Even once we have developed the duration feature to its fullest, will that field be excluded from being managed in the form configuration. This is because of https://community.openproject.org/projects/openproject/work_packages/40539
which will result in the dates and the duration to always be displayed in the work package page. But for 12.2 we can start by excluding the `:duration`.

Reply:
DONE: f73b1412f1eb1be91f4e957e25f99f96d08de206